doc/developer_doc/40-setup-cvmfs-server.md

```diff
@@ -0,0 +1,46 @@
+# Setup of CernVM-FS Server
+
+### Prerequisites
+- `cvmfs` is installed
+- `autofs` is disabled
+
+### Goal
+ - Create, modify and delete CernVM-FS repository called `local.test.repo` 
+
+**NOTE**: If you do not disable `autofs` before any of the CernVM-FS server manipulation commands you can get in a broken state which can only be resolved by restarting the entire machine! (Independent of if you later on disable `autofs`)
+
+```py
+    ####################################################
+    # CREATE repo
+    ####################################################
+    sudo cvmfs_server mkfs local.test.rep
+
+    ####################################################
+    # START MODIFY files - this is done in a transaction
+    ####################################################
+
+    # start transaction
+    sudo cvmfs_server transaction local.test.repo
+
+    # Perform the file manipulations
+    sudo cp /home/myuser/testfile*.txt /cvmfs/local.test.repo/
+    sudo rm /cvmfs/local.test.repo/testfile2.txt
+
+    # Finalize transaction
+    sudo cvmfs_server publish local.test.repo
+
+    # Abort transaction and discard all changes
+    sudo cvmfs_server abort local.test.repo
```

Comment:
This command of course would fail just after having committed a transaction.

Reply:
yes, i know. this is just more supposed to be like a list of the commands that are there and good to know. does it need clarifications for it?